app/workers/check_status_worker.rb

```diff
@@ -4,7 +4,7 @@ class CheckStatusWorker
   include Sidekiq::Worker
   sidekiq_options queue: :status, unique: :until_executed
 
-  def perform(project_id, _ignored = nil)
+  def perform(project_id)
```

Comment:
I'd suggest leaving the ignored parameter in until all callers don't send it for a bit. There's otherwise a race condition of jobs being in sidekiq that were previously enqueued and hitting a worker running new code that then throws an error